### PR TITLE
vmui: change the `query_range` request method from `GET` to `POST`

### PR DESCRIPTION
**Pull request by Loori-R**
### Describe Your Changes

change the `/query_range` and `/query` requests method from `GET` to `POST`. See #6288.

### Checklist

The following checks are **mandatory**:

- [x] My change adheres [VictoriaMetrics contributing guidelines](https://docs.victoriametrics.com/contributing/).

### CODE REVIEW

**Review by dmitryk-dk — APPROVED** [review]
LGTM!

File: docs/changelog/CHANGELOG.md [hunk 1]
```diff
@@ -18,6 +18,8 @@ See also [LTS releases](https://docs.victoriametrics.com/lts-releases/).
 
 ## tip
 
+* FEATURE: [vmui](https://docs.victoriametrics.com/#vmui): change the `query_range` request method from `GET` to `POST`. See [this issue](https://github.com/VictoriaMetrics/VictoriaMetrics/issues/6288).
```
Comment (hagen1778) [inline docs/changelog/CHANGELOG.md:21]: Does it mean that /query request will remain GET?

Comment (Loori-R) [reply docs/changelog/CHANGELOG.md]: No, the `/query` request has also been changed to POST. I've updated the changelog to reflect this correction.

**Review by hagen1778 — APPROVED** [review]
LGTM

**Pull request merged**